generators/Generators.fs

```diff
@@ -1326,6 +1326,38 @@ type Series() =
         |> Option.ofNonErrorObject
         |> Option.renderParenthesized
 
+type SgfParsing() = 
+    inherit GeneratorExercise()
+
+    override this.PropertiesWithIdentifier _ = ["expected"]
```

Comment:
One final nit (sorry about this): could you also add "input" here, to have the input also be a separate identifier?